pandas/tests/io/test_html.py

```diff
@@ -82,33 +44,56 @@ def assert_framelist_equal(list1, list2, *args, **kwargs):
         assert not frame_i.empty, 'frames are both empty'
 
 
+def _missing_bs4():
+    bs4 = td.safe_import('bs4')
+    if not bs4 or LooseVersion(bs4.__version__) == LooseVersion('4.2.0'):
```

Comment:
note that would be ok requiring a later version of bs4 to avoid some of the older issues

Reply:
Updated the min version. FWIW in `ci/requirements-2.7_COMPAT.pip` the requirement is hardcoded at `4.2.0` and `ci/requirements-2.7_LOCALE.pip` has `4.2.1`. I'm not sure the backstory to those but I assume that will cause conflict

Reply:
May be rooted back in #4259?